include/EASTL/sort.h

```diff
@@ -707,6 +754,34 @@ namespace eastl
 	}
 
 
+	template <typename RandomAccessIterator, typename T, typename Compare>
+	inline RandomAccessIterator get_partition(RandomAccessIterator first, RandomAccessIterator last, const T&& pivotValue, Compare compare, typename eastl::enable_if<eastl::is_move_constructible<T>::value>::type* = 0 )
```

Comment:
Need to upgrade my install of vim... it seems to be too old for clang-format.py :(